src/REL/d/a/obj/d_a_obj_cloud_dive.cpp

```diff
@@ -1,13 +1,115 @@
 #include "d/a/obj/d_a_obj_cloud_dive.h"
 
+#include "d/a/d_a_item.h"
+#include "d/a/d_a_player.h"
+#include "d/a/obj/d_a_obj_base.h"
+#include "d/col/cc/d_cc_s.h"
+#include "egg/math/eggMath.h"
+#include "f/f_base.h"
+#include "m/m_vec.h"
+
 SPECIAL_ACTOR_PROFILE(OBJ_CLOUD_DIVE, dAcOcloudDive_c, fProfile::OBJ_CLOUD_DIVE, 0x1C2, 0, 0);
 
+const f32 dAcOcloudDive_c::mRadius1 = 100.0f;
+const f32 dAcOcloudDive_c::mRadius2 = 200.0f;
+const f32 dAcOcloudDive_c::lbl_350_rodata_8 = 40.0f;
```

Comment:
This appears to be the height. Rename it accordingly and replace all references to `40.0f` with it (and references to `20.0f` with height / 2.0f)